### DISCUSSION TITLE
bugs

### DISCUSSION THREAD

**Discussion by radiolamp** [blackadderkate/weather-widget-2#96] [Q&A]
file:///home/anton/.local/share/plasma/plasmoids/org.kde.weatherWidget/contents/ui/main.qml:113:5: Type YrNo unavailable

file:///home/anton/.local/share/plasma/plasmoids/org.kde.weatherWidget/contents/ui/providers/YrNo.qml:18:1: module "QtQuick.XmlListModel" is not installed

archlinux kde5

**Comment by blackadderkate** [comment]
Try installing the package qt5-xmlpatterns

**Reply by AlgoTide** [reply]
debian11(stable) has not this package

**Comment by blackadderkate** [comment]
@NeformaliTu
I believe the package you need to installed is called "qml-module-qtquick-xmllistmodel".
